Add information about xpub (#2665)
rotki/rotki: docs/usage_guide.rst

If an ethereum account also contains tracked tokens you can click on the arrow under "Actions" in order to expand its view and show the balance breakdown for the account over all assets it holds.

> For Bitcoin you can add addresses manually or let rotki discover them using a xpub. From this key rotki can generate your addresses and can query the Bitcoin's blockchain for each one of them until it finds unused addresses.

Reviewer: ```suggestion
For Bitcoin you can add addresses manually or let rotki discover them using an xpub. From this key rotki can generate your addresses and query the Bitcoin blockchain for each one of them until it finds unused addresses.
```

> xpub also generates different types of addresses, P2PKH's addresses have "1" as prefix, P2SH_P2wPKH's addresses start with a "3" and addresses generated by WPKH start with "bc1". 

Reviewer: ```suggestion
There are also different types of xpubs. P2PKH xpubs generate addresses that have "1" as a prefix, P2SH_P2WPKH xpubs generate addresses that start with a "3" and WPKH xpubs generate addresses that start with "bc1". You will need to know what type of xpub your bitcoin wallet generates in order to choose the correct type at the dropdown menu. If your wallet generates an xpub prefixed with ``ypub`` or an xpub prefix with ``zpub`` rotki can deduce the type for you automatically.
```

> This key does not allow spending your coins but provides information about your wallet, in rotki all this information is encrypted and safe.

Reviewer: ```suggestion
An xpub does not allow spending your coins but provides information about your wallet. In rotki this information is stored safely encrypted in your local database.
```

If an ethereum account also contains tracked tokens you can click on the arrow under "Actions" in order to expand its view and show the balance breakdown for the account over all assets it holds.

> For Bitcoin you can add addresses manually or let rotki discover them using a public key. From this key rotki knows how to generate your addresses and can query the Bitcoin's blockchain generating them until it finds an address without transactions.

Reviewer: `a public key` -> either say `an xpub` or an `extended public key`

`key rotki knows how to` -> `key rotki can`

`can query the Bitcoin's blockchain generating them until it finds an address without transactions` -> `query the Bitcoin blockchain for each one of them until it finds unused addresses`.

> The different formats of allowed keys refer to how the addresses are generated, "P2PKH" uses keys starting with "xpub", "P2SH_P2WPKH" uses keys starting with "ypub" and "WPKH" uses keys starting with "zpub".

Reviewer: That is not correct. This is just a convention. For example ledger live WILL NOT follow this convention. While the electrum wallet will. It's a confusing mess.

This is why you as a user **have** to tell us what type of xpub it is. If it starts with zpub, or ypub you don't. We can see that it follow the convention and that's great. But for say xpubs generated by ledger live (all starting with `xpub` irrespective of type) we need this extra info.

> This public keys also generate different types of addresses, "xpub"'s addresses have "1" as prefix, "ypub"'s addresses start with a "3" and addresses generated by "zpub" key start with "bc1".  When you paste your key in the "XPUB" field rotki will automatically detect the format.

Reviewer: public keys -> xpub

Also instead of saying xpub, ypub, zpub here specify the type. P2PKH, P2SH_P2wPKH or WPKH.

Also as I explained above we don't always auto-detect the format. Only if the ypub/zpub is given. We can't otherwise. Can't assume that an xpub is always legacy (P2PKH).

> This keys don't allow to spend your coins but provide information about your wallet, in rotki all this information is encrypted and safe.

Reviewer: This keys don't allow to spend  -> this key does not allow spending
